### ISSUE TITLE
Validation data

### ISSUE THREAD

**Issue by blueMoon7** [lyft/l5kit#86]
Hi

* I have been able to download the training set. Where can I find validation and test sets?
* Are you planning to release the weights of the model that has been trained on entire dataset and the results is published in the paper?
* Is there any evaluation server or leaderboard yet?

Thanks

**Comment by lucabergamini** [comment]
Hi @blueMoon7 


> * I have been able to download the training set. Where can I find validation and test sets?

validation and (public) test set will be released the day the competition goes live



> * Are you planning to release the weights of the model that has been trained on entire dataset and the results is published in the paper?

Possibly, but only after the end of the competition



> * Is there any evaluation server or leaderboard yet?

Not yet, but will be there!
